Library/Formula/ucon64.rb

```diff
@@ -0,0 +1,37 @@
+class Ucon64 < Formula
+  desc "A ROM backup tool and emulator's Swiss Army knife program."
+  homepage "http://ucon64.sourceforge.net/"
+  url "https://downloads.sourceforge.net/ucon64/ucon64-2.0.0-src.tar.gz"
+  sha256 "62064324a1912387f84ac9d4c521c5d5e7b80f2567e9f61bf0ab3e1d976c0127"
+
+  def install
+    # ucon64's normal install process installs the discmage library in
+    # the user's home folder. We want to store it inside the prefix, so
+    # we have to change the default value of ~/.ucon64rc to point to it.
+    # .ucon64rc is generated by the binary, so we adjust the default that
+    # is set when no .ucon64rc exists.
+    inreplace "src/ucon64_misc.c", "~/.ucon64/discmage.dylib",
+                                   opt_prefix/"libexec/libdiscmage.dylib"
+
+    cd "src" do
+      system "./configure", "--disable-debug",
+                            "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{prefix}"
+      system "make"
+      bin.install "ucon64"
+      libexec.install "libdiscmage/discmage.so" => "libdiscmage.dylib"
+    end
+  end
+
+  def caveats
+    <<-EOS.undent
+      You can copy/move your DAT file collection to $HOME/.ucon64/dat
+      Be sure to check $HOME/.ucon64rc for some options after you've run uCON64 once.
+    EOS
+  end
+
+  test do
+    system "#{bin}/ucon64", "--help", "--snes"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!